package.json

```diff
@@ -31,7 +31,16 @@
   "homepage": "https://github.com/snyk/code-client#readme",
   "jest": {
     "preset": "ts-jest",
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "testTimeout": 50000,
```

Comment:
question: any reason to increase the timeout here?

Reply:
default one was too small for some test repos. Since we don't want to have flaky tests, and we don't test performance here, i wanted to give big enough timeout